images/node/Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN INSTALL_PKGS="libmnl libnetfilter_conntrack conntrack-tools openvswitch \
     yum clean all && \
     mkdir -p /usr/lib/systemd/system/origin-node.service.d /usr/lib/systemd/system/docker.service.d
 
+# Copy the cni plugins to the host file system when they are present, so that they can be shared with cri-o
+RUN if test -e /opt/cni/bin; then mkdir -p /exports/hostfs/opt/cni/bin/ && cp -r /opt/cni/bin/* /exports/hostfs/opt/cni/bin/; fi
```

Comment:
`/exports/hostfs` is a system container thing.  Every file in this directory is exported to the host at installation time, e.g. `/exports/hostfs/foo/bar` will be copied to the host from the container at `/foo/bar`